docs/DeveloperGuide.md

```diff
@@ -2,21 +2,133 @@
 
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links 
+to the original source as well}
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+The following is our class diagram for our whole project.
 
+![Local Image](UML/CLiRental.png)
+
+### Sequence diagrams
+
+1. `Adding a customer`
+
+The following sequence diagram will explain the sequence of events after the user inputs an add-user command.
+
+![Local Image](UML/AddCustomerSequence.png)
```

Comment:
Good attempt on the sequence diagrams.
However, there may be too many information within the diagram. Perhaps omit details that involve the Java API e.g. method calls to ArrayList.